bin/build.js

```diff
@@ -1,8 +1,15 @@
+"use strict";
+
 const fs = require("fs");
 const path = require("path");
-const kebabCase = require("kebab-case");
-const { version } = require("../package.json");
 const primitives = require("../index.json");
+const version = process.env.npm_package_version;
+
+function kebabCase(str) {
+  return str.replace(/[A-Z\u00C0-\u00D6\u00D8-\u00DE]/g, function (match) {
+    return "-" + match.toLowerCase();
```

Comment:
This doesn't seem quite right.  It'll replace all upper-case chars with `-{char}`

i.e. a string like `ABCD` would transform to `-A-B-C-D`.  Possible in practice we always start with a lowercase letter and never have multiple capitals in a row and this doesn't functionality produce a bug at the moment.

I might do this: `replace(/([a-z])([A-Z]+)/g, (matches) => matches[1] + "-" + matches[2].toLowerCase())

Reply:
I pulled that straight from the [`kebab-case` npm package](https://github.com/joakimbeng/kebab-case/blob/master/index.js). It's been doing exactly what I was looking for when building the package 🤷

Reply:
@danielbeardsley I think you're probably right, but that if we're assuming camel-case names, all-upper-case would be an acronym, and it's not completely unreasonable to turn that into a completely-hyphenated name for kebab-case.

@ianrohde Might be good to add a comment saying where that code came from.

Reply:
https://github.com/iFixit/core-primitives/pull/29/commits/1a423a041e4d560ab0f4e8f29f82b65916aaf5b0